lib/streamlit/runtime/app_session.py

```diff
@@ -726,6 +726,12 @@ def _create_new_session_message(
         )
         _populate_config_msg(msg.new_session.config)
         _populate_theme_msg(msg.new_session.custom_theme)
+        _populate_theme_msg(
+            getattr(
+                msg.new_session.custom_theme, config.CustomThemeCategories.SIDEBAR.value
+            ),
```

Comment:
nit: I think I would just do `msg.new_session.custom_theme.sidebar` for this since it refers to a static protobuf prop that we cannot easily change afterward because of proto compatibility (so we don't get a lot by making this aspect dynamic). That also gives us a bit better typing checks on this aspect. But no strong opionion.